app/build.gradle

```diff
@@ -8,7 +8,6 @@ apply from: 'witness.gradle'
 android {
 
 	compileSdkVersion 29
-	buildToolsVersion "29.0.2"
```

Comment:
The removal was recommended by the IDE and [is backed by the documentation since `3.1.0`](https://developer.android.com/studio/releases/gradle-plugin.html#3-1-0).